packages/grafana-schema/src/schema/dashboard/v2alpha0/dashboard.schema.cue

```diff
@@ -524,6 +529,56 @@ GridLayoutKind: {
   spec: GridLayoutSpec
 }
 
+RowGridLayoutKind: {
+  kind: "RowGridLayout"
+  spec: RowGridLayoutSpec
+}
+
+RowGridLayoutSpec: {
+  items: [...GridLayoutItemKind]
+}
+
+RowsLayoutKind: {
+  kind: "RowsLayout"
+  spec: RowsLayoutSpec
+}
+
+RowsLayoutSpec: {
+  rows: [...RowsLayoutRowKind]
+}
+
+RowsLayoutRowKind: {
+  kind: "RowsLayoutRow"
+  spec: RowsLayoutRowSpec
+}
+
+RowsLayoutRowSpec: {
+  title?: string
+  collapsed: bool
+  repeat?: RowRepeatOptions
+  layout: RowGridLayoutKind | ResponsiveGridLayoutKind
```

Comment:
The layout in a row can be either a `GridLayoutKind` or `ResponsiveGridLayoutKind`. Looking over it, I don't think we need `RowGridLayoutKind` at all.

Reply:
This is a very subtle difference and something I'm not sure how to better represent. RowGridLayoutKind is very similar to the GridLayoutKind (DefaultLayoutManager). The difference is RowGridLayoutKind does not allow the legacy rows.

Reply:
I think this is going to be the case for tabs as well. Might need a better name for it. `NewGridLayoutKind`?

Reply:
That's a good point. However adding 'New' might not be the best idea, because in like 5 years it's still going to be called new 😆

Reply:
Not sure I understand the need for RowGridLayoutKind,

Reply:
@torkelo

GridLayoutKind supports both GridLayoutItemKind and GridLayoutRowKind as items. GridLayoutRowKind is the old rows that can be used in DefaultGridLayoutManager. RowGridLayoutKind only allows GridLayoutItemKind.

I did it this way for the schema to not allow the structure row layout -> grid layout -> old style row.

I made this diagram earlier to try to explain how I think we could proceed
![image](https://github.com/user-attachments/assets/f6efbfa6-4444-445e-8c17-e454b6511977)

Reply:
but we have GridLayoutRowKind why would we need RowGridLayoutKind? 
 
> I did it this way for the schema to not allow the structure row layout -> grid layout -> old style row.

I think this just overly complicates the schema, we can prohibit that in the UI

Reply:
so I would just skip that for now, just focus on keeping it simple and understandable :) it's confusing enough with the two row layouts , we do not need a third!

Reply:
@torkelo I've removed the RowGridLayout in the latest commit :)

Reply:
![image](https://github.com/user-attachments/assets/a7a5dd93-1603-4bc8-9c1a-18fc63eb6e99)